applications/asset_tracker_v2/overlay-pgps.conf

```diff
@@ -4,6 +4,8 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+CONFIG_BUILD_WITH_TFM=n
+CONFIG_SPM=y
```

Comment:
```suggestion
# Disable TFM (not yet supported in asset_tracker_v2)
CONFIG_BUILD_WITH_TFM=n
CONFIG_SPM=y
```

Reply:
@joerchan I am unsure why this goes into the P-GPS overlay. 
Shouldn't this be disabled for all builds of `asset_tracker_v2`?

/cc @simensrostad

Reply:
@coderbyheart It is TF-M + PGPS that I'm seeing an issue with, so doing this as a temporary solution until that is fixed.